src/components/DataDimension/Groups.js

```diff
@@ -14,10 +14,11 @@ export const Groups = ({
     onDetailChange,
     onGroupChange,
 }) => {
-    const defaultGroup = dataTypes[dataType].defaultGroup
     let optionItems = groups
 
-    if (defaultGroup) {
+    if (dataTypes[dataType].defaultGroup) {
+        const { id, getName } = dataTypes[dataType].defaultGroup
+        const defaultGroup = { id, name: getName() }
         optionItems = [defaultGroup, ...optionItems]
```

Comment:
change to `optionItems = [{ id, name: getName() }, ...optionItems]`? or is `defaultGroup` used somewhere else?

Reply:
Changed, though I did have the extra const there to make the code somewhat more self explanatory.